include/lsst/afw/cameraGeom/Detector.h

```diff
@@ -150,6 +150,28 @@ class Detector {
     lsst::afw::table::AmpInfoRecord const & operator[](std::string const &name) const;
 
     /**
+     * Get the amplifier specified by index as a shared_ptr
```

Comment:
I'm very glad you pointed this out. That triggered a discussion on Slack dm-docs about enabling autobrief in doxygen, which prompted https://jira.lsstcorp.org/browse/DM-8647. Having merged that, I'm going to omit `@brief` in the new code.